es/criteria/document-objectives.md

```diff
@@ -6,25 +6,25 @@ order: 8
 
 ## Requisitos
 
-* La codebase DEBE contener documentación de sus objetivos -como una declaración de misión y objetivos- que sea comprensible para profesionales del diseño y desarrollo, para que de esta manera puedan utilizar o contribuir a la codebase.
+* La codebase DEBE contener documentación de sus objetivos - como una declaración de misión y objetivos - que sea comprensible para profesionales del diseño y desarrollo, para que de esta manera puedan utilizar o contribuir a la codebase.
```

Comment:
Spaces not necessary in Spanish: https://es.wikipedia.org/wiki/Raya_(puntuaci%C3%B3n)

If controversial, could be substituted by parenthesis, too.